lib/constructs/interface.js

```diff
@@ -10,33 +10,13 @@ const Overloads = require("../overloads");
 const Parameters = require("../parameters");
 const keywords = require("../keywords");
 
-function isPropGetter(idl) {
-  return idl.type === "operation" && idl.getter && Array.isArray(idl.arguments) && idl.arguments.length === 1;
-}
-
-function isPropSetter(idl) {
-  return idl.type === "operation" && idl.setter && Array.isArray(idl.arguments) && idl.arguments.length === 2;
-}
-
-function isPropDeleter(idl) {
-  return idl.type === "operation" && idl.deleter && Array.isArray(idl.arguments) && idl.arguments.length === 1;
-}
-
-function isNamed(idl) {
-  return idl.arguments[0].idlType.idlType === "DOMString";
-}
+// Used as a sentinel in inheritedMembers to signify that the following members are inherited.
+const kInherited = Symbol("inherited");
 
-function isIndexed(idl) {
-  return idl.arguments[0].idlType.idlType === "unsigned long";
-}
-
-function isValueIterable(idl) {
-  return idl.type === "iterable" && !Array.isArray(idl.idlType);
-}
-
-function isPairIterable(idl) {
-  return idl.type === "iterable" && Array.isArray(idl.idlType) && idl.idlType.length === 2;
-}
+/* eslint-disable func-style */
+const isNamed = idl => idl.arguments[0].idlType.idlType === "DOMString";
```

Comment:
Sure, I can change them. I just thought the one-line style is more appropriate for this type of shorthand function.